src/nxdoc/web-ui/web-ui-tutorials/how-to-use-rich-text-editor-rte-for-article-publishing.md

```diff
@@ -0,0 +1,189 @@
+    name: coverage-aggregate-designer-catalog.png
+    1.1.3#screenshot#up_to_date
+--}}
+![coverage-aggregate-designer-catalog.png](nx_asset://9fe81437-386b-4386-bf11-c9f04bd53e82 ?border=true)
+
+{{!--     ### nx_asset ###
+    path: /default-domain/workspaces/Product Management/Documentation/Documentation Screenshots/NXDOC/Master/how-to-use-rte-article-publishing/coverage-aggregate-designer-generation.png
+    name: coverage-aggregate-designer-generation.png
+    1.1.3#screenshot#up_to_date
+--}}
+![coverage-aggregate-designer-generation.png](nx_asset://afdecac0-95c2-4308-a712-6a6020373894 ?border=true)
+
+Save and [deploy your configuration]({{page page='nuxeo-dev-tools-extension'}}#hot-reload) to see the result. This time you should edit your picture to add a coverage to it, then create a version to see it appear in the list when picking it from the rich text editor.
+
+{{!--     ### nx_asset ###
+    path: /default-domain/workspaces/Product Management/Documentation/Documentation Screenshots/NXDOC/Master/how-to-use-rte-article-publishing/rte-results-with-search-filter-configured.png
+    name: rte-results-with-search-filter-configured.png
+    1.1.3#screenshot#up_to_date
+--}}
+![rte-results-with-search-filter-configured.png](nx_asset://777f47d0-2740-4e73-90a0-64ef3f57d24d ?border=true)
+
+To configure the search results listing, [configure the **Results** option]({{page page="ui-designer" space="studio"}}) in Nuxeo Studio Designer for your `document_picker` page provider instead.
+
+## Going Further
+
+The following examples will be covered from a high level perspective only.
+
+### Using Multiple Rich Text Editors
+
+Let's assume you want to add a field to put a summary of your article. Or that you want to separate the introduction, the main content of the article and the conclusion to adapt your content more easily to a wider variety of website layouts.
+
+In this kind of situation, it is possible to use multiple rich text editors and to map them to different properties of your documents. Here's a summary of how you could proceed.
+
+1. Create a custom document type.
+1. Add a custom schema to it, and add your new fields using the `String` type.
+1. In Nuxeo Studio Designer, configure the edit layout for your document type.
+1. Drag and drop a `nuxeo-html-editor` element and bind it to the corresponding field. Repeat that operation for every additional field you created.
+
+### Tracking Relations Between Documents
+
+When referencing pictures in the rich text editor, Nuxeo won't keep track of which images were used for which article. To make overall maintenance and tracking easier, you may want to implement a relation system.
+
+Here's a summary of how you could proceed. Note that implementing this feature requires coding knowledge.
+
+1. Add an event handler that will be triggered on the following events for your custom document type: `About to create`, `Before document modification`.
```

Comment:
Are there any documentation for these events that we could link here?

Reply:
Added links